pages/docs/tutorials/javascript/working-with-javascript.md

```diff
@@ -1,103 +1,21 @@
-
-
-<div class="sample" markdown="1" theme="idea" data-highlight-only>
-```kotlin
-document.bgColor = "FFAA12" 
-```
-</div>
-
-The DOM also provides us a way to retrieve a specific element by ID, name, class name, tag name and so on. All returned elements are of type `NodeList`, and to access members we need to cast them to the specific type of element. The code below shows how we could access an input
-element on the page:
-
-<div class="sample" markdown="1" theme="idea" mode="xml">
-```html
-<body>
-    <input type="text" name="email" id="email"/>
-    <script type="text/javascript" src="scripts/kotlin.js"></script>
-    <script type="text/javascript" src="scripts/domInteraction.js"></script>
-</body>
-```
-</div>
-
-<div class="sample" markdown="1" theme="idea" data-highlight-only>
-```kotlin
-val email = document.getElementById("email") as HTMLInputElement
-email.value = "hadi@jetbrains.com"
-```
-</div>
-
-An important note is to make sure that the scripts are located before the ``body`` tag is closed. Placing them at the top means that the scripts would be loaded before the DOM is fully available.
-
-Much like we reference an input element, we can access other elements on the page, casting them to the appropriate types. 
-
-## Using kotlinx.html
-
-The [kotlinx.html library](http://www.github.com/kotlin/kotlinx.html) provides the ability to generate DOM using statically typed HTML builders.
-The library is available when targeting the JVM as well as JavaScript. To use the library we need to include the corresponding
-dependency. In the case of Gradle this would be 
-
-<div class="sample" markdown="1" theme="idea" mode="groovy">
-```groovy
-repositories {
-    mavenCentral()
-    maven {
-        url  "http://dl.bintray.com/kotlin/kotlinx.html/"
-    }
-}
-
-dependencies {
-    compile 'org.jetbrains.kotlinx:kotlinx-html-js:0.6.1'
-    compile "org.jetbrains.kotlin:kotlin-stdlib-js:$kotlin_version"
-}
-```
-</div>
-
-For more information about configuring Gradle to target JavaScript please see [Getting Started with Gradle](getting-started-gradle/getting-started-with-gradle.html).
-
-Once the dependency is included, we can access the different interfaces provided to generate DOM. The code below will add a new ```span``` tag with the text ```Hello``` inside a ```div``` on the
-`window.load` event.
-
-<div class="sample" markdown="1" theme="idea" data-highlight-only>
-```kotlin
-import kotlin.browser.*
-import kotlinx.html.*
-import kotlinx.html.dom.*
-
-fun printHello() {
-    window.onload = {
-        document.body!!.append.div {
-            span {
-                +"Hello"
-            }
-        }
-    }
-}
-```
-</div>
-
 ## Using dukat to generate header files for Kotlin
```

Comment:
Do we want to write about gradle plugin integration with dukat in experimental mode?